crates/sui-protocol-config/src/lib.rs

```diff
@@ -1187,6 +1196,12 @@ impl ProtocolConfig {
                 cfg
             }
             13 => Self::get_for_version_impl(version - 1, chain),
+            14 => {
+                let mut cfg = Self::get_for_version_impl(version - 1, chain);
+                cfg.gas_rounding_step = Some(1_000);
+                cfg.gas_model_version = Some(6);
```

Comment:
this is not the same v14 that is currently in main (missing Emma's transaction ordering change)

Reply:
@emmazzz has changed that and it should be the same